sdk/storage/Azure.Storage.Files.Shares/src/Generated/Models/ShareErrorCode.cs

```diff
@@ -207,6 +208,8 @@ public ShareErrorCode(string value)
         public static ShareErrorCode ShareSnapshotOperationNotSupported { get; } = new ShareErrorCode(ShareSnapshotOperationNotSupportedValue);
         /// <summary> ShareHasSnapshots. </summary>
         public static ShareErrorCode ShareHasSnapshots { get; } = new ShareErrorCode(ShareHasSnapshotsValue);
+        /// <summary> PreviousSnapshotNotFound. </summary>
+        public static ShareErrorCode PreviousSnapshotNotFound { get; } = new ShareErrorCode(PreviousSnapshotNotFoundValue);
```

Comment:
I think you might need to run `eng/scripts/ExportApi.ps1 Storage` again, this new error code should appear in sdk/storage/Azure.Storage.Files.Shares/api/Azure.Storage.Files.Shares.net6.0.cs

Reply:
Also, where did you discover this new error code?  In the spec, or was it returned by the service?

Reply:
it was returned by the service in the error case (409)!
I will run ps1 again